core/arch/arm/dts/stm32mp13-pinctrl.dtsi

```diff
@@ -26,6 +26,19 @@
 	};
 
 	uart4_pins_a: uart4-0 {
+		pins1 {
+			pinmux = <STM32_PINMUX_NSEC('D', 6, AF8)>; /* UART4_TX */
+			bias-disable;
+			drive-push-pull;
+			slew-rate = <0>;
```

Comment:
UART4 is not securable on stm32mp13x platforms.

Note: fix s/USART4/UART4 in commit header